arch/sim/src/sim/up_hosttime.c

```diff
@@ -89,3 +91,42 @@ void host_sleepuntil(uint64_t nsec)
       usleep((nsec - now) / 1000);
     }
 }
+
+/****************************************************************************
+ * Name: host_setup_timer
+ *
+ * Description:
+ *   Set up a timer to send periodic signals.
+ *
+ ****************************************************************************/
+
+int host_setup_timer(void)
```

Comment:
Change to host_setimer to align with other function naming?

Reply:
I will update the function name